optimism/src/keccak/folding.rs

```diff
@@ -29,8 +29,10 @@ pub(crate) struct KeccakConfig;
 
 impl FoldingColumnTrait for KeccakColumn {
     fn is_witness(&self) -> bool {
-        // All Keccak columns are witness columns
-        true
+        match self {
+            KeccakColumn::Selector(_) => false,
```

Comment:
What's the semantics of this trait, what's exactly a "witness" for folding? Selectors are part of the computation, so why are they not part of the witness? Because they're public?..

Reply:
Well, they will be considered aside so as to avoid duplicates. But that will come after optimizations.